### DISCUSSION TITLE
how to change a winding current value via pyaedt

### DISCUSSION THREAD

**Discussion by iaskgithub** [ansys/pyaedt#4245] [General]
the windingA has been  created in ansys EM ,and I just to change it's current value by pyedt:
https://github.com/ansys/pyaedt/blob/9b2610d5d86a6594c37662cf0080576a752851dc/pyaedt/application/Analysis.py#L2342-L2411
anyone can give me a example to change the  current value of created winding ?

**Comment by IreneWoyna** [comment]
Hello @iaskgithub, 

One possible solution is this:

Windings = m3d.boundaries_by_type["Winding Group"]
Winding1 = Windings[0]
Winding1.props["Current"] = "10A"
Winding1.props["Phase"] = "0deg"

I hope this helps. If not, please let me know.

Thanks
Irene
